web/templates/note/index.html.eex

```diff
@@ -0,0 +1,10 @@
+<h1>Listing Notes</h1>
+
+<table class="table">
```

Comment:
This is how the book chose to render the user list in the Rumbl app. I don't have enough experience to make a judgement one way or the other, so if you think a list makes more sense I'll go with that. 

A table seems more sensible to me when I consider that I might want to add columns (for example, what if I wanted to add the date on the right hand side?). I'm interested in hearing your thought process about where to use tables vs. lists now.